lib/reporter.js

```diff
@@ -151,8 +155,9 @@ class CucumberReporter {
     handleAfterScenarioEvent (event, callback) {
         const scenario = event.getUri ? event : event.getPayloadItem('scenario')
         this.emit('suite:end', {
+            uid: this.getUniqueIdentifier(scenario),
             title: scenario.getName(),
-            parent: this.runningFeature.getName(),
+            parent: this.getUniqueIdentifier(this.runningFeature),
```

Comment:
Why do we need to change this? I am wondering if a reporter picks up this value it would have the line number without a space character in it, so something like `Foo bar34` which would look weird. Better would be `Foo Bar (line 34)`.